Test_Automation_Framework/Epam_TestAutomation_API_Tests/API3_PhonesTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Epam_TestAutomation_API;
+using Epam_TestAutomation_API.Controllers;
+using Epam_TestAutomation_API.RequesModelsPhone;
+using Epam_TestAutomation_API.ResponseModels;
+using Epam_TestAutomation_API.SharedModelsPhone;
+using RestSharp;
+using NUnit.Framework;
+
+namespace Epam_TestAutomation_API_Tests
+{
+    [TestFixture]
+    public class API3_PhonesTests
+    {
+        private string deleteThePhone;
+
+        [Test]
+        public void CheckTheCapacityOfTheCreatedObjectTest()
+        {
+            var addPhone = new PhonesModels
+            {
+                name = "Apple iPhone 12 Pro Max",
+                data = new PhonesData
+                {                  
+                    color = "Cloudy White",
+                    CapacityGb = 512
+                }
+            };
+
+            var createPhone = new PhoneController(new CustomRestClient()).AddPhone<AllPhonesModels>(addPhone).Phone;
+            var receivePhone = new PhoneController(new CustomRestClient()).GetPhonesID<AllPhonesModels>(createPhone.id).Phone;
+            deleteThePhone = receivePhone.id;
```

Comment:
I Like the fact that you passed this as variable. But it would be better to place not string, but the whole model into this property to work with.